[2023-09-03T08:34:41.213+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T08:34:27.490288+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-03T08:34:41.214+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T08:34:27.490288+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T08:34:27.490288+00:00/task_id=create_listings_raw_table'
[2023-09-03T08:34:41.249+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T08:34:27.490288+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-03T08:34:41.250+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T08:34:27.490288+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T08:34:27.490288+00:00/task_id=create_listings_raw_table'
[2023-09-03T08:34:41.275+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table manual__2023-09-03T08:34:27.490288+00:00 [queued]>
[2023-09-03T08:34:41.285+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table manual__2023-09-03T08:34:27.490288+00:00 [queued]>
[2023-09-03T08:34:41.286+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-09-03T08:34:41.301+0000] {taskinstance.py:1382} INFO - Executing <Task(BigQueryCreateEmptyTableOperator): create_listings_raw_table> on 2023-09-03 08:34:27.490288+00:00
[2023-09-03T08:34:41.307+0000] {standard_task_runner.py:57} INFO - Started process 3314 to run task
[2023-09-03T08:34:41.310+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Autotrader_ETL_DAG', 'create_listings_raw_table', 'manual__2023-09-03T08:34:27.490288+00:00', '--job-id', '166', '--raw', '--subdir', 'DAGS_FOLDER/autotrader_etl.py', '--cfg-path', '/tmp/tmp7zgs3vb3']
[2023-09-03T08:34:41.312+0000] {standard_task_runner.py:85} INFO - Job 166: Subtask create_listings_raw_table
[2023-09-03T08:34:41.367+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T08:34:27.490288+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-03T08:34:41.368+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T08:34:27.490288+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T08:34:27.490288+00:00/task_id=create_listings_raw_table'
[2023-09-03T08:34:41.371+0000] {task_command.py:415} INFO - Running <TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table manual__2023-09-03T08:34:27.490288+00:00 [running]> on host 1a0e8b05e04d
[2023-09-03T08:34:41.463+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Autotrader_ETL_DAG' AIRFLOW_CTX_TASK_ID='create_listings_raw_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-03T08:34:27.490288+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-03T08:34:27.490288+00:00'
[2023-09-03T08:34:41.476+0000] {connection.py:232} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2023-09-03T08:34:41.480+0000] {base.py:73} INFO - Using connection ID 'google_cloud' for task execution.
[2023-09-03T08:34:41.481+0000] {bigquery.py:1507} INFO - Creating table
[2023-09-03T08:34:42.532+0000] {bigquery.py:1535} INFO - Table autotrader_staging.listings_raw already exists.
[2023-09-03T08:34:42.567+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Autotrader_ETL_DAG, task_id=create_listings_raw_table, execution_date=20230903T083427, start_date=20230903T083441, end_date=20230903T083442
[2023-09-03T08:34:42.609+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-03T08:34:42.642+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
